common/app/templates/inlineJS/blocking/cloudwatchBeacons.scala.js

```diff
@@ -1,6 +1,18 @@
 @(item: model.MetaData)(implicit request: RequestHeader)
 @import conf.switches.Switches._
 
+@if(OfflinePageView.isSwitchedOn) {
+    try {
+        var offlineViews = parseInt(localStorage.getItem('gu.offlineViews'));
+        for (var i = 0; i < offlineViews; i++) {
+            (new Image()).src = window.guardian.config.page.beaconUrl + '/count/offline-page-view.gif';
```

Comment:
I *think* that's what `analytics/Metric.scala` does? iv'e fallen foul of that before, maybe not